src/journey-maps/angular/services/map/map-stopover-service.ts

```diff
@@ -0,0 +1,19 @@
+import { Injectable } from '@angular/core';
+import { FeatureCollection } from 'geojson';
+import { GeoJSONSource, Map as MaplibreMap } from 'maplibre-gl';
+
+import { ROKAS_STOPOVER_SOURCE } from '../constants';
+
+import { SBB_EMPTY_FEATURE_COLLECTION } from './map-service';
+
+@Injectable({ providedIn: 'root' })
+export class SbbMapStopoverService {
```

Comment:
Purpose of having this as a separate class, instead of extending an existing, related class?

Reply:
I removed this class again